src/legacy/core_plugins/console/public/src/components/help_show_panel.tsx

```diff
@@ -17,26 +17,31 @@
  * under the License.
  */
 
-require('./sense_help_example');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { I18nContext } from 'ui/i18n';
+import { OpenHelpPanel } from './help_panel';
```

Comment:
Nit: Can we remove the verb from this name by changing `OpenHelpPanel` to `HelpPanel`? I'm used to seeing verbs used in method/function names, and nouns in classes and class instances. :)

Reply:
No idea why I did this!
Think I copied it from somewhere O_O